File: javascript/1073.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var i=parseInt(lines[0])
var c;
for(let j=1;j<=i;j++){
    if(j%2 == 0){
      c=Math.pow(j,2) 
      console.log(`${j}^2 = ${c}`)
    }
}

File: C/1073.c
#include <stdio.h>
#include <math.h>
int main() {
 
    int i,c;
    scanf("%i",&i);
    for(int j=1;j<i+1;j++){
        if(j%2==0){
            c=pow(j,2);
            printf("%i^2 = %i\n",j,c);
        }
    }
    return 0;
}